backend/database.js

```diff
@@ -36,32 +37,49 @@ if (!fs.existsSync(dataFolder)) {
   fs.mkdirSync(dataFolder, { mode: 0o750 })
 }
 
+// Streams stored contract log entries since the given entry hash (inclusive!), most recent first.
```

Comment:
Just like with `eventsBefore` API, clients should be able to specify the number of `eventsAfter`, so please add an extra parameter to this function to take that number in (and update `routes.js` as well).

If the number isn't passed in, then use `MAX_EVENTS_AFTER`.